event_session/models/event_session.py

```diff
@@ -114,9 +109,9 @@ class EventSession(models.Model):
         store=True,
     )
     seats_available_pc = fields.Float(
-        string="Full %", readonly=True, compute="_compute_seats",
+        string="Full %", store=True, readonly=True, compute="_compute_seats",
```

Comment:
Is there a motivation to change this field to stored?

Reply:
Yes, there was. Runbot failed with a warning. because `_compute_seats` was used for stored fields but `seats_available_pc`. The error was "inconsistent compute_sudo for computed fields", a v13 feature I think. Maybe an alternative for this field should be forcing `compute_sudo=True` instead of making it stored, and warning should be avoided too. What do you think?

Reply:
Please use `compute_sudo=True` then.

Reply:
Done!